### PR TITLE
DOI badge

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

**Comment by dpo** [comment]
@abelsiqueira do you have an ORCID id?

**Comment by abelsiqueira** [comment]
Yes: https://orcid.org/0000-0003-4451-281X

**Review by abelsiqueira — APPROVED** [review]
Thanks for the update.

File: CITATION.cff [hunk 1]
```diff
@@ -1,10 +1,10 @@
 cff-version: 1.2.0
 message: "If you use this software, please cite it as below."
 authors:
-  - family-names: Template
-    given-names: JSO
+  - family-names: Siqueira
+    given-names: Abel
     orcid: https://orcid.org/0123-4567-89AB-CDEF
```
Comment (dpo) [inline CITATION.cff:6]:
```suggestion
    orcid: https://orcid.org/0000-0003-4451-281X
```

**Pull request merged**